draft-ietf-gnap-core-protocol.md

```diff
@@ -5372,11 +5375,8 @@ have their own tradeoffs discussed elsewhere in this section.
 TLS functions at the socket layer, ensuring that only the parties on either end of that socket
 connection can read the information passed along that connection. Each time a new socket connection
 is made, such as for a new HTTP request, a new trust is re-established that is unrelated to previous
-connections. As such, it is not possible with TLS alone to know that the same party is making
-a set of calls, and therefore TLS alone cannot provide the continuity of security needed
-for GNAP. However, mutual TLS (MTLS) does provide such security characteristics through the
-use of the TLS client certificate, and thus MTLS is acceptable as a key-presentation mechanism
-when applied as described in {{mtls}}.
+connections. It is not possible with TLS alone to know that the same party is making
+a set of calls. Hence the methods {{binding-keys}} described in GNAP.
```

Comment:
```suggestion
connections. In other words, it is not possible with TLS alone to know that the same party is making
a set of calls over time, since each time a new TLS connection is established, both the client and the server (or the server only when using {{mtls}}) have to validate
the other party's identity. Such a verification can be achieved via methods described in {{!I-D.ietf-uta-6125bis}}), but these are not enough to establish the identity of the client instance in many cases.

To counter this, GNAP defines a set of key binding methods in {{binding-keys}} that allow authentication and
proof of possession by the caller, which is usually the client instance. These methods are intended to be used in
addition to TLS on all connections.

```